content/docs/pipelines/reference/api/kubeflow-pipeline-api-spec.html

```diff
@@ -3082,1104 +3801,1344 @@ <h3 class="panel-title"><span class="operation-name">POST</span> <strong>/apis/v
-                                                    
+
                                                 </dt>
                                                 <dd>
                                                     <p>Output. Unique pipeline ID. Generated by API server.</p>
-                                    
+
                                                     <div class="json-inner-schema">
-                                                        
+
                                                     </div>
                                                 </dd>
                                                 <dt data-property-name="created_at">
                                                     <span class="json-property-name">created_at:</span>
-                                                    
+
                                                     <span class="json-property-type">string</span>    <span class="json-property-format">(date-time)</span>
                                                     <span class="json-property-range" title="Value limits"></span>
-                                                    
+
                                                 </dt>
                                                 <dd>
                                                     <p>Output. The time this pipeline is created.</p>
-                                    
+
                                                     <div class="json-inner-schema">
-                                                        
+
                                                     </div>
                                                 </dd>
                                                 <dt data-property-name="name">
                                                     <span class="json-property-name">name:</span>
-                                                    
+
                                                     <span class="json-property-type">string</span>
                                                     <span class="json-property-range" title="Value limits"></span>
-                                                    
+
                                                 </dt>
                                                 <dd>
                                                     <p>Optional input field. Pipeline name provided by user. If not specified,
                                     file name is used as pipeline name.</p>
-                                    
+
                                                     <div class="json-inner-schema">
-                                                        
+
                                                     </div>
                                                 </dd>
                                                 <dt data-property-name="description">
                                                     <span class="
```

Comment:
s/id/ID

Reply:
Also, how about replacing "not empty" with "present" in both cases?

Reply:
Added for the second comment, unsure what is intended by the first.